src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -141,69 +161,91 @@
 rescale_after_resolution_cutoff = False
   .help = "Re-scale the data after application of a resolution cutoff"
   .type = bool
+  .short_caption = "Rescale after resolution cutoff"
 
-filtering {
+filtering
+  .short_caption = "Filtering"
+{
 
   method = None deltacchalf
     .type = choice
     .help = "Choice of whether to do any filtering cycles, default None."
 
-  deltacchalf {
+  deltacchalf
+    .short_caption = "ΔCC½"
+  {
     max_cycles = None
       .type = int(value_min=1)
+      .short_caption = "Maximum number of cycles"
     max_percent_removed = None
       .type = float
+      .short_caption = "Maximum percentage removed"
     min_completeness = None
       .type = float(value_min=0, value_max=100)
       .help = "Desired minimum completeness, as a percentage (0 - 100)."
+      .short_caption = "Minimum completeness"
     mode = dataset image_group
       .type = choice
       .help = "Perform analysis on whole datasets or batch groups"
     group_size = None
       .type = int(value_min=1)
       .help = "The number of images to group together when calculating delta"
               "cchalf in image_group mode"
+      .short_caption = "Group size"
     stdcutoff = None
       .type = float
       .help = "Datasets with a ΔCC½ below (mean - stdcutoff*std) are removed"
+      .short_caption = "Standard deviation cutoff"
   }
 }
 
 multi_crystal_analysis {
   include scope xia2.Modules.MultiCrystal.master_phil_scope
 }
 
-unit_cell {
+unit_cell
+  .short_caption = "Unit cell"
+{
   refine = *two_theta
     .type = choice(multi=True)
 }
 
-two_theta_refine {
+two_theta_refine
+  .short_caption = "2θ refinement"
+{
   combine_crystal_models = True
     .type = bool
+    .short_caption = "Combine crystal models"
 }
 
 min_completeness = None
   .type = float(value_min=0, value_max=1)
+  .short_caption = "Minimum completeness"
 min_multiplicity = None
   .type = float(value_min=0)
+  .short_caption = "Minimum multiplicity"
 max_clusters = None
   .type = int(value_min=1)
+  .short_caption = "Maximum number of clusters"
 cluster_method = *cos_angle correlation unit_cell
   .type = choice
+  .short_caption = "Descriptor on which to perform clustering"
```

Comment:
Metric rather than descriptor?
```suggestion
  .short_caption = "Metric on which to perform clustering"
```